src/App/App.tsx

```diff
@@ -33,10 +33,26 @@ class App extends Component<IProps, IState> {
 	}
 
 	setAppGenre = (genre: string) => {
-		this.setState({ selectedGenre: genre })
-		getPlaylist(this.state.selectedGenre)
+		this.setState({ selectedGenre: genre });
+		const splitGenreArray = this.parseGenreForFetch(genre);
+		getPlaylist(splitGenreArray)
 		.then(data => this.setState({ playlists: [...this.state.playlists, [...data]] }));
 	}
+
+	parseGenreForFetch = (genre: string) => {
+		const fillerWordsToRemove = /(the |and |of |\/| |ism)+/;
+		const parseGenreArray = genre.split(fillerWordsToRemove);
+		if (parseGenreArray.includes('')) {
```

Comment:
This `if` statement could be refactored out if there is a way to exclude the empty string that gets left in the last index of an array if splitting by the final character of a string.

I'll look more into `.split()` but for now it seems like a code limitation.